src/main/scala/org/apache/spark/sql/hbase/util/HBaseKVHelper.scala

```diff
@@ -128,6 +128,21 @@ object HBaseKVHelper {
   }
 
   /**
+   * Takes a record, translate it into HBase row key column matching with metadata
+   * @param values record that as a sequence of string
+   * @param keyBytes  output parameter, array of (key column and its type);
+   */
+  def string2Key(values: Seq[String],
+                lineBuffer: Array[BytesUtils],
+                keyColumns: Seq[AbstractColumn],
+                keyBytes: Array[(Array[Byte], DataType)]) = {
```

Comment:
only in createSplitKeys we use this method, right? if so i think we no need create this method in HBaseKVHelper